.github/actions/common-versioning/action.yml

```
@@ -78,12 +78,18 @@ runs:
           echo "Full version will be: ${FULL_VERSION}"
 
-          # Set outputs using printf to ensure no trailing newlines
+          # Set all outputs first, before any debug output
           {
-            printf "version=%s\n" "${UPDATED_VERSION}"
-            printf "full_version=%s\n" "${FULL_VERSION}"
+            echo "version=${UPDATED_VERSION}"
+            echo "full_version=${FULL_VERSION}"
+            echo "short_sha=${SHORT_SHA}"
           } >> "$GITHUB_OUTPUT"
           
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)